### DISCUSSION TITLE
What binary sensor should be used

### DISCUSSION THREAD

**Discussion by KaherdinTristan** [fuatakgun/eufy_security#83] [Q&A] [answered]
Hi
I have several different EufyCams. But this one is the T8142-Z. 
The binary sensor that "should be used" to trigger a live feed on a Card, as I understand it, is normally
"binary_sensor.<NAME>_streaming_sensor" right? 

The thing is that, that binary sensor is ALWAYS set to "True". 
- Is this an error, or am I missing some setting somewhere. 
All help is appreciated
Peace 🕊️

**Comment by fuatakgun** [comment]
Please dump attributes of your camera

**Comment by fuatakgun** [comment]
i feel like your rtsp stream is always on? So it is normal to have this binary sensor as True all the time.

**Reply by KaherdinTristan** [reply]
Ok, but is there a binary sensor that only triggers on movement or when the app would play live? 
The dump contains a lot of private info, so I removed it...

**Reply by fuatakgun** [reply]
Please check device entities, there are bunch of them

**Comment by KaherdinTristan** [comment, marked as answer]
My Security settings are set to HOME. I don't want to set the Alarm Security Settings to AWAY. 
Is there any way to achieve Live view on motion, without setting the Alarm Settings to AWAY? 

All these binary sensors are True. 
![image](https://user-images.githubusercontent.com/3398594/139300133-921f44b9-a1c7-40a3-b7a0-588120333466.png)

**Reply by fuatakgun** [reply]
live view is not related to security settings but event pictures are related.
you can always start a live stream (using either RTSP or P2P) and see live video.
but if you want to get pictures of captured events, you should enable push notification in security settings.
